funnel/models/user.py

```diff
@@ -188,6 +188,28 @@ def name(cls):  # NOQA: N805
     def is_active(self):
         return self.status == USER_STATUS.ACTIVE
 
+    @property
+    def is_comment_moderator(self):
+        from .site_membership import SiteMembership
+
+        return (
+            SiteMembership.query.filter_by(
+                user=self, is_comment_moderator=True, is_active=True
+            ).one_or_none()
+            is not None
+        )
+
+    @property
+    def is_user_moderator(self):
+        from .site_membership import SiteMembership
+
+        return (
+            SiteMembership.query.filter_by(
+                user=self, is_user_moderator=True, is_active=True
+            ).one_or_none()
+            is not None
+        )
```

Comment:
We can stick these in from site_membership.py. Avoids import inside the function.